### PR TITLE
mypy_primer: add to CI

### PR DESCRIPTION
**Pull request by hauntsaninja**

### CODE REVIEW

File: stdlib/3/functools.pyi [hunk 1]
```diff
@@ -22,10 +22,6 @@ _AnyCallable = Callable[..., Any]
 
 _T = TypeVar("_T")
 _S = TypeVar("_S")
-@overload
```
Comment (JelleZijlstra) [inline stdlib/3/functools.pyi:25]: Looks accidental?

Comment (hauntsaninja) [reply stdlib/3/functools.pyi:25]: Oh, this was so that you could see what failure looks like. (Unlike Python) I very much plan on adding reduce back :-)

**Comment by hauntsaninja** [comment]
I'm going to go ahead and merge this; we'll see how it does in practice

**Comment by JukkaL** [comment]
This is great! This should help catch some regressions that would otherwise would be easy to miss.

**Pull request merged**